whois-endtoend/src/test/java/net/ripe/db/whois/query/InverseQueryTestIntegration.java

```diff
@@ -216,6 +216,35 @@ public void inverse_mnt_ref_no_results() {
 
         assertThat(response, containsString("no entries found"));
     }
+
+    @Test
+    public void inverse_sponsoring_org_then_succeed() {
+        databaseHelper.addObject(RpslObject.parse("" +
+                "organisation: ORG-SPONSOR\n" +
+                "org-name:     Sponsoring Org Ltd\n" +
+                "org-type:     LIR\n" +
+                "descr:        test org\n" +
+                "address:      street 5\n" +
+                "e-mail:       org1@test.com\n" +
+                "mnt-ref:      OWNER-MNT\n" +
+                "mnt-by:       OWNER-MNT\n" +
+                "source:       TEST\n" +
+                ""));
+        databaseHelper.addObject("" +
+                "aut-num:        AS102\n" +
+                "as-name:        End-User-2\n" +
+                "descr:          description\n" +
+                "sponsoring-org: ORG-SPONSOR\n" +
+                "admin-c:        PP1-TEST\n" +
+                "tech-c:         PP1-TEST\n" +
+                "mnt-by:         OWNER-MNT\n" +
+                "source:         TEST\n");
+
+        final String response = query("-i sponsoring-org ORG-SPONSOR");
+
+        assertThat(response, containsString("ORG-SPONSOR"));
```

Comment:
Can you check for the aut-num `AS102` also in the response

Reply:
Done